app/src/main/java/org/zephyrsoft/trackworktime/model/WeekRowState.java

```diff
@@ -0,0 +1,18 @@
+package org.zephyrsoft.trackworktime.model;
+
+import androidx.annotation.NonNull;
+
+public class WeekRowState {
+
+	@NonNull public String date, in, out, worked, flexi;
+	{
+		date = in = out = worked = flexi = "";
+	}
```

Comment:
Maybe that's oldschool, but could you do this in the "traditional" java bean style?
```
@NonNull
public String date = "";
@NonNull
public String in = "";
@NonNull
public String out = "";
@NonNull
public String worked = "";
@NonNull
public String flexi = "";
```
Or even make the variables private and let the IDE generate getters and setters?

Reply:
Kind of prefer my own style, since I think it has less duplication involved, but I don't mind changing it, for the sake of consistency with other data classes.

Do you mind if I use `Lombok` for getter and setter generation?

Reply:
I'd prefer to stay away from Lombok. I have to use it at work, and it complicates matters (1.) when setting up a new development environment (in our case: potential new fellow developers could get disappointed if they set up Android Studio, clone the repo and nothing works) and (2.) when debugging through code which uses Lombok-generated code. It's usable and removes boilerplate code, I don't want to diminish that, but it comes at a cost.